pkgs/applications/networking/wgnord/default.nix

```diff
@@ -13,13 +13,13 @@
 
 resholve.mkDerivation rec {
   pname = "wgnord";
-  version = "0.2.0";
+  version = "0.2.1";
 
   src = fetchFromGitHub {
     owner = "phirecc";
     repo = pname;
     rev = version;
-    hash = "sha256-00513jr3Sk8YahqI14pQ7pU4P7MBUlsqXXfrACu35RQ=";
+    hash = "sha256-26cfYXtZVQ7kIRxY6oNGCqIjdw/hjwXhVKimVgolLgk=";
   };
 
   postPatch = ''
```

Comment:
The patch should include a substitution for the line added in the new release:
https://github.com/phirecc/wgnord/blob/fcdb5a3c72bfd58e56e8ab214d8f8db08f8c246c/wgnord#L26

Guessing something like the below would suffice:
```
--replace '$conf_dir/auth_token' "$out/share/auth_token"
```